pertpy/tools/_perturbation_space/_discriminator_classifiers.py

```diff
@@ -149,32 +148,48 @@ def load(  # type: ignore
         batch_size: int = 256,
         test_split_size: float = 0.2,
         validation_split_size: float = 0.25,
-    ):
-        """Creates a classifier model and dataloaders required for training and testing.
+        max_epochs: int = 40,
```

Comment:
```suggestion
        max_epochs: int = 10,
```

Isn't that quite a high default?

Reply:
We have early stopping implemented, so it shouldn't matter that much. As soon as the model starts to overfit, training will be stopped anyway. Probably we could still decrease the epoch number and also the `val_epochs_check` based on what we have seen for the various datasets so far.

Reply:
Oh yeah, that's fair.